### ISSUE TITLE
ConvNeXt poor results

### ISSUE THREAD

**Issue by fedllanes** [open-mmlab/mmclassification#996]
I've been trying to use several models on my custom dataset, getting usually around 50% accuracy on the first epoch and then it goes up until 66%.

However with ConvNeXt i get around 15-20% acc on the first epoch and then it only goes up to 40%. 

Apparently ConvNeXt models are the ones that perform best on ImageNet, I don't understand why my results are so bad right now.

**Comment by Ezra-Yu** [comment]
What is the size of your data? Have you used the pre-trained model to fine-tune?

**Comment by fedllanes** [comment]
> What is the size of your data? Have you used the pre-trained model to fine-tune?

Yes, I've been using a pre-trained model for both cases that's why it's so weird to me. It seems like it's training for scratch. 
I got around 9k images for 7 classes.

**Comment by qmcreeper** [comment]
I have trained more than 300 epochs with ConvNeXt.At begining, the train loss much more bigger than hrnet.But when lr goes small, the loss goes down more quickly, like cosin plot.Maybe change lr or epochs in config can help you.

**Comment by tonysy** [comment]
We will update the configuration, which can be used to reproduce the classification result #1256

**Comment by kamzero** [comment]
> We will update the configuration, which can be used to reproduce the classification result #1256

Hi! Could you please tell me what effect does the original ConvNeXt implementation have on downstream tasks, as you mentioned in this commit?
